util/dvsim/RdcCfg.py

```diff
@@ -15,6 +15,8 @@ class RdcCfg(LintCfg):
 
     def __init__(self, flow_cfg_file, hjson_data, args, mk_config):
 
+        self.waves = args.waves or ""
```

Comment:
just out of curiosity, could CDC use this as well?

if so, it probs makes sense to add `self.waves = args.waves or ""` to that class as well (I am not suggesting to do the TCL setup yet, since that needs more work).

Reply:
I didn't check CDC manual yet. But I think CDC probably has similar setting. Adding this to CDC Cfg then use the environment variable inside `run-cdc.tcl` seems to be doable.